src/mdacli/__init__.py

```diff
@@ -1,3 +1,17 @@
+#!/usr/bin/env python3
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+#
+# Copyright (c) 2021 Authors and contributors
+#
+# Released under the GNU Public Licence, v2 or any higher version
+# SPDX-License-Identifier: GPL-2.0-or-later
 """Command line interfaces for MDAnalysis analysis classes."""
 
+from .cli import maincli, setup_clients
+
 __version__ = '0.0.0'
+
+
+def main():
```

Comment:
I would maintain this in the `cli.py`. I don't recognize this kind of functionality as being suitable for `__init__.py`.

Reply:
I'm fine with leaving it in the cli :)